.travis.yml

```diff
@@ -28,6 +28,23 @@ jobs:
         # submit coverage.txt to codecov.io
         - bash <(curl -s https://codecov.io/bash)
 
+    # YAML alias, for settings shared across the tests on macOS
+    - &osx-test
+      stage: Test
```

Comment:
is osX tests doing anything different then the linux tests?

Reply:
> is osX tests doing anything different then the linux tests?

No it is just running the unit test on mac but we don't want https://github.com/openshift/odo/pull/2549/files#diff-354f30a63fb0907d4ad57269548329e3L29 to be done in mac test. This was becoming duplication earlier as mentioned by @amitkrout in comment https://github.com/openshift/odo/pull/2549#issuecomment-580772298 .

Again we want to trigger a job for macOS where unit, integration and e2e test will run. So just created a separate template for osx. Like we are using one for linux.